src/user/entities/user.entity.ts

```diff
@@ -44,4 +45,7 @@ export class User {
 
   @OneToMany(() => UserToken, (userToken) => userToken.user)
   tokens: UserToken[];
+
+  @OneToMany(() => UserSession, (userSession) => userSession.user)
```

Comment:
Tokens are actual tokens (used for e.g. user activation and, in the near future, password reset), whereas the session is the actual session identifier. I did not want to mix tokens with the session, because they are different entities in my opinion.